### ISSUE TITLE
Sort group stats by key or by value

### ISSUE THREAD

**Issue by DragonBanana** [mattrighetti/group-activity-bot-rs#1]
Hi @mattrighetti,
First of all, I'd like to say that I really like `GroupActivityBot`. However, when sending the command `/groupstats@gactivitybot`, the group stats result is returned in a random order.
I was wondering if you can fix this issue.

A possible solution could be to use BTrees which are known for having sorted keys.
Another possible solution is to sort the keys whenever they are used:
Instead of:
```rust
impl PrettyPrint for HashMap<String, f32> {
    fn pretty_print(&self) -> String {
        let mut builder = String::new();
        for (k, v) in self {
            builder.push_str(format!("{}: {:.2}%\n", k, v).as_str());
        }
        
        builder
    }
}
```
we could use this implementation with Itertools which extends the standard iterator so that it can be sorted:
```rust
use itertools::Itertools;  // itertools = "0.8"

impl PrettyPrint for HashMap<String, f32> {
    fn pretty_print(&self) -> String {
        let mut builder = String::new();
        for key in self.keys().sorted() {
          builder.push_str(format!("{}: {:.2}%\n", k, self[key]).as_str());
        }

        builder
    }
}
```

P.S I think it could be a nice idea if you change the main branch from `master` to `main`.
```
"I really appreciate the name change [because] it raises awareness.There are things that we accept as implicit, and we then realize that we can change them because they don't match our society."
Javier Cánovas
```

**Comment by lterrac** [comment]
Hello, 
I also have another implementation enabling sorting by values of the `HashMap`. I will coordinate with @DragonBanana in order to open a PR that will include both sorting strategies.

P.s. I strongly agree with @DragonBanana  regarding the branch name. It should not take too much time to change the branch name.

**Comment by mattrighetti** [comment]
Submit a PR and I'll review it 😄

**Comment by mattrighetti** [comment]
Fixed in a72044f18aee167be22cab33d6dac0243edcef45
